src/index.js

```diff
@@ -72,8 +82,10 @@ const QRCode = ({
   color = 'black',
   backgroundColor = 'white',
   logo,
+  logoSVG,
   logoSize = size * 0.2,
   logoBackgroundColor = 'transparent',
+  logoColor = "#000000",
```

Comment:
Shouldn't we provide `transparent` here?

Reply:
this is the fill color of the svg, so I thought by default black makes more sense, because transparent means nothing is visible. Transparent makes total sense for bg I think.
Actually `null` or `undefined` will be better I think to simply not set it at all if it's not provided.

I will check if this gives good results

Reply:
Yeah, not setting it makes the most sense since you won't override the default then.

Reply:
fixed